pxtlib/tilemap.ts

```diff
@@ -816,6 +816,9 @@ namespace pxt {
                 case AssetType.Image:
                     return this.state.images.getByID(name) || this.gallery.images.getByID(name);
                 case AssetType.Tile:
+                    if (name == "myTiles.tile0") {
```

Comment:
i thought transparency was named "myTiles.transparency16" or whatever the tile size is? or did it get renamed during the refactor process?

Reply:
oh yes the old way was myTiles.tile0